src/fontra/client/core/glyph-controller.js

```diff
@@ -525,6 +525,7 @@ export class StaticGlyphController {
     this.errors = errors;
     this.canEdit = layerName != undefined;
     this.components = [];
+    this.anchors = instance.anchors || [];
```

Comment:
I don't think this is needed.

Reply:
It does not work without it. Either I do something wrong or it's needed.

Reply:
Ah, I see it. You need to define a getter:
```python
  get anchors() {
    return this.instance.anchors;
  }
```

Reply:
Thanks. Done.